src/components/Home/Home.js

```diff
@@ -39,7 +40,9 @@ export default class Home extends React.Component {
   
   hidemovie = () => {
     this.state.effect.from('.home', 
-    { ease: Back.easeOut, x: 2990, duration: 1.5})
+    { ease: Back.easeOut, x: 2990, duration: 1.5}).then(() => {
+      document.querySelector('.home').removeAttribute('style')
```

Comment:
good job on figuring that you can use .then method after animation to remove the style atr!!